src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/GoogleConnectPage.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+
+/**
+ * Created by Ludwik on 2015-11-04.
+ */
+public class GoogleConnectPage extends WikiBasePageObject {
+
+  @FindBy(css = "#message")
+  private WebElement message;
+
+  @FindBy(css = ".abcRioButtonContentWrapper")
+  private WebElement SignUpButton;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "SignUpButton" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)